modules/virt-enlarging-vm-disk.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virtual_disks/virt-vm-disk-enlargement.adoc
+
+[id="virt-enlarging-vm-disk_{context}"]
+= Enlarging a virtual machine disk
+
+A virtual machine (VM) disk has a default or initial storage capacity when you create it. However, you can enlarge the disk by resizing the disk's persistent volume claim (PVC).
```

Comment:
This first sentence doesn't contribute a lot of meaning on its own. My guess is that you can enlarge a VM disk more than once, too, so you might not always be enlarging it from its original state. I'd suggest rephrasing this to something like:

`You can enlarge a virtual machine (VM) disk by resizing the disk's persistent volume claim (PVC).`